### PR TITLE
update readme w/ session_pool_size

### PR DESCRIPTION
**Pull request by sjoeboo**
update readme w/ session_pool_size example

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -216,6 +223,7 @@ The constructors of `GrafanaApi` and `GrafanaClient`, as well as the factory met
 scalar `float` value, or as a tuple of `(<read timeout>, <connect timeout>)`.
 
 
+```
```
Comment (amotl) [inline README.md:226]: Was this change intended? Maybe it has been placed on a wrong line?
```suggestion
```
```

File: README.md [hunk 2]
```diff
@@ -206,6 +206,13 @@ environment variables. Setting them before invoking an application using
 export HTTP_PROXY=10.10.1.10:3128
 export HTTPS_PROXY=10.10.1.11:1080
 ```
+### Session Pool Size
+
+By default a session pool size of 10 is used. This can be changed by passing the
+`session_pool_size` argument to the `GrafanaApi` constructor:
+
+```python
+grafana.client.session_pool_size = 32
```
Comment (amotl) [inline README.md:215]: Ah, I think ``` should be placed here instead.

```suggestion
```python
grafana.client.session_pool_size = 32
```

```

Comment (sjoeboo) [reply README.md:215]: oops, fixing

Comment (amotl) [reply README.md:215]: Fixed already. Thanks.
